socksio/socks5.py

```diff
@@ -97,12 +98,38 @@ def loads(cls, data: bytes) -> "SOCKS5UsernamePasswordReply":
         return cls(success=data == b"\x01\x00")
 
 
-class SOCKS5Request(typing.NamedTuple):
+class SOCKS5CommandRequest(typing.NamedTuple):
     command: SOCKS5Command
     atype: SOCKS5AType
     addr: bytes
     port: int
 
+    @classmethod
+    def from_address_port(
+        cls, command: SOCKS5Command, address: str, port: typing.Optional[int] = None
+    ) -> "SOCKS5CommandRequest":
+        """Convenience method for creating command requests from
+        standard address strings in the form of '127.0.0.1:3080'
+        """
+        if port is None:
+            try:
+                address, str_port = address.split(":")
+                port = int(str_port)
+            except ValueError as exc:
+                raise ValueError(
+                    "Port missing, please supply a port integer manually"
+                    "or include it in the address parameter as a suffix"
+                    "`127.0.0.1:3080`"
+                ) from exc
```

Comment:
Let's change to `from None` since we don't want the original traceback, unless we do want the traceback?

Reply:
Not really, I just got into the habit of doing that 👍